x/concentrated-liquidity/types/math.go

```diff
@@ -1,14 +1,40 @@
-package concentrated_liquidity
+package types
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v12/osmomath"
 )
 
+// tickToSqrtPrice takes the tick index and returns the corresponding sqrt of the price
+func TickToSqrtPrice(tickIndex sdk.Int) (sdk.Dec, error) {
```

Comment:
Why did we move this here?

These seem to be core math functions that shouldn't be exported outside of the module

Reply:
I think for methods that does not use Ctx, there's no need to be inside the (keeper) or the core module, instead can be at types, so that it can be accessed more easily from different packages. What's the reason you think it should be in the core module?